pyk/src/pyk/prelude/kint.py

```diff
@@ -34,3 +34,79 @@ def geInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
 
 def eqInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
     return KApply('_==Int_', i1, i2)
+
+
+def neqInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    return KApply('_=/=Int_', i1, i2)
```

Comment:
Please add docstrings to all these functions. Here's a template:

```suggestion
def neqInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
    """Instantiate the KAST term ```_=/=Int_`(i1, i2)``.

    Args:
        i1: The left operand.
        i2: The right operand.

    Returns:
        The KAST term ``\`_=/=Int_\`(i1, i2)``.
    """
    return KApply('_=/=Int_', i1, i2)
```

Renders like this:

![image](https://github.com/runtimeverification/k/assets/10085903/bc65b428-1fbd-4f1d-a639-05d131fb7f92)

(Run `make pydocstyle` to check docstring format. Run `make docs` and open `docs/build/index.html` in a browser to check the result.)